camel/agents/chat_agent.py

```diff
@@ -134,8 +131,7 @@ def __init__(
         model_config: Optional[Any] = None,
```

Comment:
`chat_agent` is also used for other models. Maybe Changing to `Optional[ChatGPTConfig]` is not an good idea.

Reply:
In general, Any should not be there. Here we could make an empty ABC base class for ChatGPTConfig and other model configs. But a better solution would be injecting a pre constructed model backend. This will replace `model` and `model_config` and improve code cohesion. @lightaime

Reply:
I agree.